osu.Game/Screens/Backgrounds/BackgroundScreenBeatmap.cs

```diff
@@ -37,15 +50,19 @@ public WorkingBeatmap Beatmap
                         }
 
                         b.Depth = newDepth;
-                        AddInternal(Background = b);
+                        FadeContainer.Add(Background = b);
                         Background.BlurSigma = BlurTarget;
+                        FadeContainer.StoryboardReplacesBackground.BindTo(StoryboardReplacesBackground);
                     }));
                 });
             }
         }
 
         public BackgroundScreenBeatmap(WorkingBeatmap beatmap = null)
         {
+            FadeContainer = CreateFadeContainer();
+            InternalChild = FadeContainer;
```

Comment:
Same thing here, this line could just be `InternalChild = FadeContainer = CreateFadeContainer()`.